src/features/user/TreeMapper/Import/components/PlantingLocation.tsx

```diff
@@ -41,6 +49,99 @@ const dialogSx: SxProps = {
   },
 };
 
+interface SpeciesProps {
+  index: number;
+  t: Function;
+  remove: Function;
+  errors: FieldErrors<Record<number, any>>;
+  item: Record<'id', string>;
+  control: Control<any, any>;
+}
+
+function PlantedSpecies({
+  index,
+  t,
+  remove,
+  errors,
+  item,
+  control,
+}: SpeciesProps): ReactElement {
+  return (
+    <div key={item.id} className={styles.speciesFieldGroup}>
+      <div className={styles.speciesNameField}>
+        {/* <SpeciesSelect label={t('treemapper:species')} name={`plantedSpecies[${index}].species`} mySpecies={mySpecies} control={control} /> */}
+        <Controller
+          name={`plantedSpecies[${index}].otherSpecies`}
+          control={control}
+          rules={{
+            required:
+              index > 0 ? false : t('treemapper:atLeastOneSpeciesRequired'),
+          }}
+          render={({ field: { onChange, onBlur, value } }) => (
+            <MaterialTextField
+              label={t('treeSpecies')}
+              variant="outlined"
+              onChange={onChange}
+              onBlur={onBlur}
+              value={value}
+            />
+          )}
+        />
+        {errors.plantedSpecies &&
+          errors.plantedSpecies[index]?.otherSpecies && (
+            <span className={styles.errorMessage}>
+              {errors.plantedSpecies[index]?.otherSpecies &&
+                errors.plantedSpecies[index]?.otherSpecies.message}
+            </span>
+          )}
```

Comment:
In 3409146fb9e8540b3b7c4fca7d11c90ee8da5f31, we'd replaced TextField with MaterialTextField and used the TextField props to show errors. Any reason to reverse that change? This may be done inadvertently while merging from develop